spring-cloud-sleuth-core/src/main/java/org/springframework/cloud/sleuth/instrument/web/HttpTags.java

```diff
@@ -0,0 +1,72 @@
+/*
+ * Copyright 2013-2016 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.springframework.cloud.sleuth.instrument.web;
+
+/**
+ * Well-known {@link org.springframework.cloud.sleuth.Span#tag(String, String) span tag} keys.
+ */
+public final class HttpTags {
+
+  /**
+   * The domain portion of the URL or host header.
+   *
+   * <p>Used to filter by host as opposed to ip address.
+   */
+  public static final String HTTP_HOST = "http/host";
+
+  /**
+   * The HTTP method, or verb, such as "GET" or "POST".
+   *
+   * <p>Used to filter against an http route.
+   */
+  public static final String HTTP_METHOD = "http/method";
+
+  /**
+   * The absolute http path, without any query parameters.
+   *
+   * <p>Used to filter against an http route, portably with zipkin v1.
+   *
+   * <p>In zipkin v1, only equals filters are supported. Dropping query parameters makes the number
+   * of distinct URIs less. For example, one can query for the same resource, regardless of signing
+   * parameters encoded in the query line. This does not reduce cardinality to a HTTP single route.
+   * For example, it is common to express a route as an http URI template like
+   * "/resource/{resource_id}". In systems where only equals queries are available, searching for
+   * {@code http.uri=/resource} won't match if the actual request was "/resource/abcd-ff".
+   */
+  public static final String HTTP_URI = "http/uri";
+
+  /**
+   * The entire URL, including the scheme, host and query parameters if available.
+   *
+   * <p>Combined with {@link #HTTP_METHOD}, you can understand the fully-qualified request line.
+   *
+   * <p>Note: This is not logged in as many systems as {@link #HTTP_URI}. Proceed with caution as
+   * the URL may include private data encoded as query parameters, and be considerably larger than
+   * other fields.
+   */
+  public static final String HTTP_URL = "http/url";
+
+  /**
+   * The HTTP response code, such as 500.
+   *
+   * <p>Used to filter for error status.
```

Comment:
note: in brave, this is only logged for non-2xx range. I think this makes sense, as it is only interesting in error case.